src/test/java/com/github/wovnio/wovnjava/QueryUrlLanguagePatternHandlerTest.java

```diff
@@ -28,15 +28,17 @@ public void testRemoveLang__NonMatchingQuery__DoNotModify() {
         QueryUrlLanguagePatternHandler sut = new QueryUrlLanguagePatternHandler();
         assertEquals("/", sut.removeLang("/", "ja"));
         assertEquals("/page/index.html", sut.removeLang("/page/index.html", "ja"));
+        assertEquals("?wovn=en", sut.removeLang("?wovn=en", "ja"));
         assertEquals("/page/?wovn=en", sut.removeLang("/page/?wovn=en", "ja"));
         assertEquals("ja.site.com/ja/?lang=ja", sut.removeLang("ja.site.com/ja/?lang=ja", "ja"));
+        assertEquals("http://site.com/page/?wovn=japan", sut.removeLang("http://site.com/page/?wovn=japan", "ja"));
     }
 
     public void testRemoveLang__MatchingQuery__RemoveLangCode() {
         QueryUrlLanguagePatternHandler sut = new QueryUrlLanguagePatternHandler();
         assertEquals("", sut.removeLang("?wovn=ja", "ja"));
         assertEquals("/?search=pizza&lang=ja", sut.removeLang("/?search=pizza&wovn=ja&lang=ja", "ja"));
-        assertEquals("site.com/page/index.html?wovn&wovn", sut.removeLang("site.com/page/index.html?wovn&wovn=ja&wovn", "ja"));
```

Comment:
Adjusted this test case because no test did the form `path/index.html?some=query?wovn=en` with the lang query at the end.